### DISCUSSION TITLE
The inclusion of Symbols along with punctuation and numbers

### DISCUSSION THREAD

**Discussion by modothprav** [monkeytypegame/monkeytype#1557] [Ideas] [answered]
Is there currently a feature to include symbols similar to the inclusion of punctuation and numbers? The feature will really help improve my typing speed while coding.

**Comment by unrealapex** [comment]
Try using the code languages.

**Comment by Miodec** [comment, marked as answer]
You can use the code languages. Those will include more special characters when punctuation mode is enabled. You can also use the specials or ascii funboxes to train special characters exclusively.

**Reply by modothprav** [reply]
Cheers thank you so much
